iteration/iteration.go

```diff
@@ -58,13 +59,22 @@ type GormIterationRepository struct {
 	db *gorm.DB
 }
 
+func ConvertToLtreeFormat(uuid string) string {
```

Comment:
Good point @kwk , I referred this [PR](https://github.com/almighty/almighty-core/pull/655/files#diff-6bf7307e9c503123fa4c0ad08e0545eeR191) for that method.
My logic behind keeping that parameter as string is - it allowed me to make sure ParentPath is safe to enter while creattion [here](https://github.com/almighty/almighty-core/pull/713/files#diff-742f09758d56dbffa579f147b30a2be5R76).
If it accepts only UUID then I need to change above line to 
```
u.ParentPath = u.ParentPath + "." + ConvertToLtreeFormat(u.ID)
```
Here, I need to be dependent on u.ParentPath to be a valid c-locale only string. And that can be coming from outside(e.g> writing test [here](https://github.com/almighty/almighty-core/pull/713/files#diff-097e95c5b6df90d05214ba7897848eafR163))
But you have a valid point too, should I change parameter name to uuidStr ? And function is one liner hence I haven't added any comment.